Dockerfile

```diff
@@ -1,11 +1,23 @@
 FROM python:3.11.9-slim
-RUN apt update
+
+RUN mkdir -p .data/results && \
+    apt-get update && \
+    apt-get install -y gcc libssl-dev && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    pip install --upgrade poetry
+    
+
 WORKDIR /usr/src/owaspnettacker
-COPY . .
-RUN mkdir -p .data/results
-RUN apt-get update
-RUN apt-get install -y gcc libssl-dev
-RUN pip3 install --upgrade poetry
-RUN python -m poetry install
+
+COPY .data .data
+COPY nettacker nettacker
+COPY nettacker.py nettacker.py
+COPY poetry.lock poetry.lock
+COPY pyproject.toml pyproject.toml
+
```

Comment:
@arkid15r  README.md also needs to be copied

Reply:
Done.